scalingo.json

```diff
@@ -0,0 +1,7 @@
+{
+  "env": {
+    "IS_REVIEW_APP": {
+      "value": "true"
+    }
+  }
```

Comment:
Wouldnt that mean the value is always at `true` even on non PR/branch-based deploys?

Reply:
I thought so but… no. This file is only processed on review apps.

> The manifest is an optional file named scalingo.json you can add at the root of your repository to configure [review apps](https://doc.scalingo.com/platform/app/review-apps) and [one-click deploy button](https://doc.scalingo.com/platform/deployment/one-click-deploy). Its intent is to configure how an application should be created.

https://doc.scalingo.com/platform/app/app-manifest